src/javascripts/button_state.js

```diff
@@ -10,7 +11,7 @@
 	input = document.getElementById('input');
 
 	// Only create the event listener if the element exists within DOM
-	if (typeof (btnLoad) != 'undefined' && btnLoad != null) {
+	if (typeof (btnLoad) !== 'undefined' && btnLoad !== null) {
 		// Create event listener for 'return' keydown and send it as click event
```

Comment:
Jake mentioned the useful comment below yesterday:

If the button is false, 0, NaN, or "", btnLoad won't be undefined or null, which will execute this function unnecessarily. It might be worth just using if(btnLoad).